cmake/external/abseil-cpp.patch.txt

```diff
@@ -0,0 +1,27 @@
+diff --git a/absl/meta/type_traits.h b/absl/meta/type_traits.h
+index d886cb30..de0ebda2 100644
+--- a/absl/meta/type_traits.h
++++ b/absl/meta/type_traits.h
+@@ -35,6 +35,12 @@
+ #ifndef ABSL_META_TYPE_TRAITS_H_
+ #define ABSL_META_TYPE_TRAITS_H_
+ 
++// Added by firestore-ios-sdk/cmake/external/abseil-cpp.patch.txt.
++#if __APPLE__
```

Comment:
Drive-by comment: I'd recommend doing an ifdef for clang rather than apple:

```
#if __clang__
```

https://clang.llvm.org/docs/LanguageExtensions.html

Reply:
OK, that'll definitely catch Xcode though?

Reply:
Yes, it absolutely should.